src/custom/state/orders/actions.ts

```diff
@@ -32,6 +32,7 @@ export interface BaseOrder extends Omit<OrderCreation, 'signingScheme'> {
   creationTime: string // Creation time of the order. Encoded as ISO 8601 UTC
   isCancelling?: boolean // Intermediate state while the order has been cancelled but order is still pending
   isUnfillable?: boolean // Whether the order is out of the market, due to price movements since placement
+  amountByCurrentPrice?: string // Amount calculated for the current market price if the order price differs from the previous one
```

Comment:
I am a bit confused by this name.

Maybe `amountForCurrentMarketPrice` better describes it? Still unsure

Reply:
I have changed it to `currentMarketPriceAmount`, WDYT?